.github/workflows/nightly-tests.yml

```diff
@@ -166,7 +166,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: powdr-labs/openvm-reth-benchmark
-          ref: main
+          ref: update-openvm
```

Comment:
Undo after https://github.com/powdr-labs/openvm-reth-benchmark/pull/25 is merged.